src/common/logging.cpp

```diff
@@ -78,12 +78,17 @@ void otbrLogSetLevel(otbrLogLevel aLevel)
 }
 
 /** Initialize logging */
-void otbrLogInit(const char *aIdent, otbrLogLevel aLevel, bool aPrintStderr)
+void otbrLogInit(const char *aProgramName, otbrLogLevel aLevel, bool aPrintStderr)
 {
-    assert(aIdent);
+    const char *ident;
+
+    assert(aProgramName != nullptr);
     assert(aLevel >= OTBR_LOG_EMERG && aLevel <= OTBR_LOG_DEBUG);
 
-    openlog(aIdent, (LOG_CONS | LOG_PID) | (aPrintStderr ? LOG_PERROR : 0), OTBR_SYSLOG_FACILITY_ID);
+    ident = strrchr(aProgramName, '/');
+    ident = (ident != nullptr) ? ident + 1 : aProgramName;
```

Comment:
Is it possible to have more than one `/` in the `aProgramName`? 
We can consider skipping over them all until we see no more.

Reply:
Hi abtink, it's exactly what [strrchr](https://cplusplus.com/reference/cstring/strrchr/) does!